tests/nnfw_api/src/ValidationTestAddSessionPrepared.cc

```diff
@@ -80,4 +80,12 @@ TEST_F(ValidationTestAddSessionPrepared, neg_set_output_002)
             NNFW_STATUS_ERROR);
 }
 
+TEST_F(ValidationTestAddSessionPrepared, neg_load_model)
+{
+  // Invalid state
+  ASSERT_EQ(nnfw_load_model_from_file(
+                _session, NNPackages::get().getModelAbsolutePath(NNPackages::ADD).c_str()),
+            NNFW_STATUS_ERROR);
```

Comment:
This also seems to load model twice.

Reply:
Yes, and also it is invalid state, since it is PREPARED state.

Reply:
It seems to return an error whether or not it is PREPARED state. I think it's a negative test of `ValidationTestAddModelLoaded`, but not a negative test of `ValidationTestAddSessionPrepared`.

Reply:
User could call this function in any state, so I just wanted to test this when it is PREPARED as well as MODEL_LOADED.

Reply:
Okay. If so, It is "load model twice", but not "Invalid state"

Reply:
"load model twice" is one of "invalid state"s. Being more specific is good so I updated the comment.